res/res.xml

```diff
@@ -12,3 +12,8 @@
 		</downloader>
 	</member_data>
 </member>
+<member name="res_pjsip_config_sangoma" displayname="Download the Sangoma PJSIP Configuration Module for Asterisk. See https://downloads.digium.com/pub/telephony/res_pjsip_config_sangoma.">
+	<support_level>external</support_level>
+	<conflict>no_binary_modules</conflict>
+	<defaultenabled>no</defaultenabled>
```

Comment:
You need the depends for xmlstarlet and bash like res_digium_phone because the download script is dependent on them.